frontend/app/views/archival_objects/_form_container.html.erb

```diff
@@ -67,7 +67,7 @@
 
   <% form.emit_template("archival_object") %>
 
-  <%= render_aspace_partial :partial => "shared/subrecord_form", :locals => {:form => form, :name => "titles"} %>
+  <%= render_aspace_partial :partial => "shared/subrecord_form", :locals => {:form => form, :name => "titles", :required => true} %>
```

Comment:
Titles for AOs are actually "conditionally" required (required only if there is no date provided), which is something I had forgotten to look into. The old title field uses `:required => :conditionally`, but I don't know if there are additional mechanisms that need to be updated for that.